packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -314,3 +330,90 @@ export function validateFieldConfig(config: FieldConfig) {
     config.min = tmp;
   }
 }
+
+const getLinksSupplier = (
+  frame: DataFrame,
+  field: Field,
+  fieldScopedVars: ScopedVars,
+  replaceVariables: InterpolateFunction,
+  options: {
+    theme: GrafanaTheme;
+  }
+) => (config: ValueLinkConfig): LinkModelSupplier<Field> => {
+  return {
+    getLinks: (scopedVars?: any): Array<LinkModel<Field>> => {
```

Comment:
what is the passed in scoped vars here?

Reply:
is not all scoped vars already part of replaceVariables or the field

Reply:
they are, removing